Classes/KIFUITestActor.m

```diff
@@ -321,6 +321,23 @@ - (void)tapAccessibilityElement:(UIAccessibilityElement *)element inView:(UIView
         KIFTestWaitCondition(view.isUserInteractionActuallyEnabled, error, @"View is not enabled for interaction: %@", view);
 
         CGPoint tappablePointInElement = [self tappablePointInElement:element andView:view];
+
+        // If the element isn't immediately tappable, try checking if it is contained within scroll views that can be scrolled to make it tappable.
+        if (isnan(tappablePointInElement.x)) {
+            UIView *container = view;
+
+            do {
+                if ([container isKindOfClass:UIScrollView.class]) {
+                    UIScrollView *containerScrollView = (UIScrollView *)container;
+                    CGRect rect = [view convertRect:view.frame toView:containerScrollView];
+                    [containerScrollView scrollRectToVisible:rect animated:NO];
+                }
+
+                container = container.superview;
+            } while (container != nil);
```

Comment:
Do you never break out of this? It seems like you're going to keep looping. Unless there's an implicit return in the call to `[containerScrollView scrollRectToVisible:animated:]`?

Reply:
`container = container.superview;` will eventually hit the top of the view hierarchy and it's superview will be nil.

Reply:
So no matter what, we always want to run through the element's entire view hierarchy, calling `[containerScrollView scrollRectToVisible:animated:]` on everything we can? We're okay with potentially calling that multiple times?

Reply:
Yep, we'll call it on all of the scroll views up to the root of the hierarchy. If you have a scrollview within a scrollview, all of them should be positioned such that the element to be tapped on should scroll to be visible.